src/get-started/mobile-gaming-example.md

```diff
@@ -285,218 +192,86 @@ It also lets the pipeline include relevant **late data**—data events with vali
+**Figure 4:** Score data for three users. Each user's scores are grouped together in a single global window, with a trigger that generates a snapshot for output ten minutes after data is received.
 
 As processing time advances and more scores are processed, the trigger outputs the updated sum for each user.
 
 The following code example shows how `LeaderBoard` sets the processing time trigger to output the data for user scores:
 
 ```java
-/**
- * Extract user/score pairs from the event stream using processing time, via global windowing.
- * Get periodic updates on all users' running scores.
- */
-@VisibleForTesting
-static class CalculateUserScores
-    extends PTransform<PCollection<GameActionInfo>, PCollection<KV<String, Integer>>> {
-  private final Duration allowedLateness;
-
-  CalculateUserScores(Duration allowedLateness) {
-    this.allowedLateness = allowedLateness;
-  }
-
-  @Override
-  public PCollection<KV<String, Integer>> expand(PCollection<GameActionInfo> input) {
-    return input.apply("LeaderboardUserGlobalWindow",
-        Window.<GameActionInfo>into(new GlobalWindows())
-            // Get periodic results every ten minutes.
-            .triggering(Repeatedly.forever(AfterProcessingTime.pastFirstElementInPane()
-                .plusDelayOf(TEN_MINUTES)))
-            .accumulatingFiredPanes()
-            .withAllowedLateness(allowedLateness))
-        // Extract and sum username/score pairs from the event data.
-        .apply("ExtractUserScore", new ExtractAndSumScore("user"));
-  }
-}
-```
-
-Note that `LeaderBoard` uses an accumulating trigger for the user score calculation (by invoking `.accumulatingFiredPanes` when setting the trigger). Using an accumulating trigger causes the pipeline to accumulate the previously emitted data together with any new data that's arrived since the last trigger fire. This ensures that `LeaderBoard` a running sum for the user scores, rather than a collection of individual sums.
+{% github_sample /apache/beam/blob/master/examples/java8/src/main/java/org/apache/beam/examples/complete/game/LeaderBoard.java tag:DocInclude_ProcTimeTrigger
+%}```
+```py
+{% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/complete/game/leader_board.py tag:processing_time_trigger
+%}```
+
+Note that `LeaderBoard` uses an accumulating trigger for the user score calculation (by <span class="language-java">invoking `.accumulatingFiredPanes`</span> <span class="language-py">using `accumulation_mode=trigger.AccumulationMode.ACCUMULATING`</span> when setting the trigger). Using an accumulating trigger causes the pipeline to accumulate the previously emitted data together with any new data that's arrived since the last trigger fire. This ensures that `LeaderBoard` is a running sum for the user scores, rather than a collection of individual sums.
```

Comment:
Here's another thing that isn't your doing - what is an "accumulating trigger"? We should find a way to phrase that differently.

Reply:
I made some changes to this so it talks about window accumulation mode. PTAL and see if it improves things!